docs/openapi/components/schemas/common/BankAccountHolderVerification.yml

```diff
@@ -0,0 +1,48 @@
+type: object
+title: Bank Account Holder Verification
+description: Verification that an entity holds a bank account as supporting evidence in for example KYC processes.
+$linkedData:
+  term: BankAccountHolderVerification
+  '@id': https://w3id.org/traceability#BankAccountHolderVerification
+properties:
+  evidenceVerifier:
+    title: Evidence Verifier
+    description: Evidence verifier.
+    $ref: ./Organization.yml
+    $linkedData:
+      term: evidenceVerifier
+      '@id': https://w3id.org/traceability#evidenceVerifier
+  bankAccountHolderVerificationMethod:
```

Comment:
I really don't like this name...

Reply:
Ahh, "VerificationMethod"... Yea, that has other connotations. I'll find a better name...